exercises/concept/log-parser/.meta/design.md

```diff
@@ -0,0 +1,30 @@
+# Design
+
+We assume that the student already knows basic regular expressions. The goal of this exercise is to teach them Elixir-specific details only.
+
+## Learning objectives
+
+- Know about the Regex module
+- Know about the `=~` operator
+- Know that some String functions accept regular expressions, e.g. match?, replace, split.
+- Know about modifiers (e.g. unicode, case-insensitive)
+- Know how to get a value from a captured group
+- Know that sigils can be used with different delimiters
```

Comment:
Good point. It would also be possible to do an analyzer check for the delimiter because that information is kept in the AST:
```
iex(2)> quote do
...(2)> ~r()
...(2)> end
{:sigil_r, [delimiter: "(", context: Elixir, import: Kernel],
 [{:<<>>, [], [""]}, []]}
```